Update code-splitting.md (#3847)
webpack/webpack.js.org: src/content/guides/code-splitting.md

  };
```

#### `optimization.runtimeChunk`

> `optimization.runtimeChunk: "single"` is needed when multiple entry points are put on a single HTML page.

Reviewer: ```suggestion
`optimization.runtimeChunk: 'single'` is needed when multiple entry points are being used on a single HTML page.
```

> Using multiple entrypoints per page should be avoided when possible in favor of an entrypoint with multiple imports: `entry: { page: ["./analytics", "./app"] }`. This results in better optimization and consistent execution order when using async script tags.

Reviewer: ```suggestion
Using multiple entrypoints per page should be avoided when possible in favor of an entrypoint with multiple imports: `entry: { page: ["./analytics", "./app"] }`. This results in a better optimization and consistent execution order when using `async` script tags.
```

> Multiple entrypoints per page are still valid to use in scenarios where HTML is generated in a dynamic matter, e. g. when components on page are unknown at compile-time and HTML page is composed dynamically depending on data.

Reviewer: ```suggestion
T> Multiple entrypoints per page could be used in scenarios where HTML is generated in a dynamic matter, e. g. when components on page are unknown at compile-time and HTML page is composed dynamically depending on the data.
```

> 

Reviewer: ```suggestion
```
